athena-jdbc/src/test/java/com/amazonaws/athena/connectors/jdbc/manager/JdbcMetadataHandlerTest.java

```diff
@@ -223,13 +223,30 @@ public void doGetTableCaseInsensitive()
     }
 
     @Test(expected = RuntimeException.class)
-    public void doGetTableCaseInsensitiveNoTablesFound()
+    public void doGetTableCaseInsensitiveDuplicateTableNames()
             throws Exception
     {
         TableName inputTableName = new TableName("testSchema", "testtable");
-        Object[][] values1 = {{"testSchema", "a"}, {"testSchema", "b"}};
+        String[] columnNames = new String[] {"table_name"};
+        String[][] tableNameValues = new String[][]{new String[] {"testTable"}, new String[] {"TestTable"}};
+        ResultSet resultSetName = mockResultSet(columnNames, tableNameValues, new AtomicInteger(-1));
+        String sql = "SELECT table_name, lower(table_name) FROM information_schema.tables WHERE (table_name = 'testTable' or lower(table_name) = 'testTable') AND table_schema = 'testSchema' LIMIT 1";
```

Comment:
qq, can you check if test working correctly? I see mockito checking limit 1 here but in our code base we remove that